src/include/sof/drivers/sai.h

```diff
@@ -251,7 +251,7 @@
  */
 #ifdef CONFIG_IMX8ULP
 /* frame clk is 16kHz on 8ulp */
-#define SAI_CLOCK_DIV		0xB
+#define SAI_CLOCK_DIV		0x17
```

Comment:
I have a feeling this isn't precise? 0xB * 2 = 0x16.

Approving either way since I'm not sure. But this is worth checking.

Reply:
According to the doc it should be bclk = mclk/((SAI_CLOCK_DIV+1)*2).

Reply:
Ah fair game then.